src/components/Register.tsx

```diff
@@ -10,8 +27,30 @@ function Register() {
     const [password, setPassword] = useState('');
     const [isRegistering, setIsRegistering] = useState(false);
 
+    const collectionRef = collection(firestore, 'users');
+
     const onSubmit = async (e) => {
         e.preventDefault();
+
+        // Firestore user POST/ADD
+        const newUser = {
+            displayName: username,
+            email: email,
+            profile_picture: 'None',
+            wishlist: [],
+            id: uuidv4(),
```

Comment:
Looks great! I think we might need the personal collection here as well, but I think we can add that later once the profile page is set up so that we can add any other features that come up in one update!